src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,12 +1,12 @@
+Microsoft.CodeAnalysis.Diagnostics.Telemetry.AnalyzerTelemetryInfo.Concurrent.get -> bool
```

Comment:
Consider reverting to avoid future merge conflicts, as this appears to just be moving the lines.

Reply:
Not addressed?

Reply:
Should be addressed now.